index.html

```diff
@@ -8,20 +8,68 @@
   <title>Awesome Books</title>
 </head>
 <body>
-  <div id="form">
-  <h1>Awesome books</h1>
-	<div class="books"></div>
-  <hr>
-  <div  id="new_book"><h1>Add new book</h1></div>
+  <header>
+    <a id="logo" href="#">Awesome books</a>
+    <ul>
+      <li>
+        <a id = "list" href="#">List</a>
+      </li>
+            <li>
+        <a id = "New" href="#">Add new</a>
+      </li>      
+      <li>
+        <a id="cont" href="#">Contact</a>
+      </li>      
+    </ul>
+    
+  </header>
+  <div  class="tim">
+  <span id="date" class="date"></span>
+  </div>
+  <main id="form">
+  <h1 class="h_books">All Awesome Books</h1>
+
+	<div class="books l_books">
+  </div>
   <br>
-	<div>
+  <div class="list" id="new_book"></div>
+  
+  <br>  
+	<div class="add_new hide">
+    <h1 class ="h_add_new">Add a new book</h1>
+
 		<input type="text" id="title" name="title" placeholder="Title">
+    <br>
 		<input type="text" id="author" name="author" placeholder="Author">
+    <br>
 		<button type="button" class="add-book-btn">Add</button>
+    <br>
+    <br>
+    <br>
```

Comment:
The _add new_ page needs some adjustments:
- [ ] The title _Add a new book_ is missing in your Add New page.
- [ ] Adjust the fonts form layout, dimensions, and positioning on of the form. Make the form bigger and centered in the page, as can be seen in the wireframe template

**Your Add New page:**
![image](https://user-images.githubusercontent.com/12745474/134706018-4e74e8b8-7509-4638-a7b2-25f96c4f21fa.png)


**The wireframe Add New page:**
![image](https://user-images.githubusercontent.com/12745474/134705307-2184f998-9c02-4279-ac18-7de5d37e50ab.png)

Reply:
Thanks for your feedback, request changes are resolved now.

Reply:
- [ ] Great Job here, however after adding the book make sure that the input fields become empty to let the user enter another book
![rr16](https://user-images.githubusercontent.com/55635977/134768789-19da4144-5516-42c5-b755-eb50a4cb9f98.png)